net/WebSocketHandler.hpp

```diff
@@ -669,10 +668,10 @@ class WebSocketHandler : public ProtocolHandlerInterface
     /// 0 for closed socket, and -1 for other errors.
     int sendMessage(const char* data, const size_t len, const WSOpCode code, const bool flush) const
     {
-        if (!Util::isFuzzing())
+        if (UnitBase::isUnitTesting() && !Util::isFuzzing())
```

Comment:
We shouldn't need to check `isUnitTesting()`, because the default UnitBase object has no-op implementation. But perhaps we don't want to instantiate them on Mobile? Is that the idea here?

Reply:
@Ashod ah - quite possibly we are just failing to do the no-op instantiation on mobile - *but* bear in mind that we have both Kit and WSD in the same process - so probably this one global that is both of these is a problem (or not). If you could dig out the code to do the no-op initialization for Patrick that would be much appreciated - Thanks ! =)

Reply:
`UnitBase::init()` has a fallback case (when no unit-lib path is given) that creates a `UnitWSD` or `UnitKit` instance.

E.g.:
```
            rememberInstance(UnitType::Wsd, new UnitWSD("UnitWSD"));
```

Reply:
I would vote against this for a couple of reasons:

1. We would probably need to pull more Online server code into the Xcode build which will likely bring in more code that doesn't build or run on iOS. AFAICT, there no tests for iOS at this time so sounds like a lot of work for no immediate gain.
2. iOS is a very resource constrained environment. Adding a bunch of code that just do nothing on iOS sounds like a very bad idea given how much effort was spent in March to make the iOS app not crash due to out of memory errors and performance bottlenecks.

By all means, if someone implements unit tests that can on iOS, go ahead and add them but can you do that in a separate pull request? Feel free to add me as a reviewer and I can see if it has any problems building or running.